README.md

```diff
@@ -160,6 +160,22 @@ Following template parameters are by default set, you can overide them by passin
 
 `Resource` returned is a map which will contains all objects of template with key `Resource Kind` with one additional key `tag` value equal to `RELEASE_VERSION`
 
+### loadResources
+
+An API to read the `resources` form yaml file. This API read all resources from the file and returns it in the form array where every array element is a key value pair of resource `kind` and `resource` itself.
```

Comment:
How about: 

### loadResources

`loadResources` returns a list of openshift `resources` read from a yaml file.